tests/postgres_tests/test_unaccent.py

```diff
@@ -42,6 +43,24 @@ def test_unaccent_chained(self):
             ordered=False
         )
 
+    def test_unaccent_chained_with_conforming_strings_off(self):
+        """
+        Chained unaccent causes the comparison string to be escaped.
+        Make sure the escaping SQL is valid even if the DB has standard_conforming_strings OFF.
+        """
+
+        with connection.cursor() as cur:
+            cur.execute("SET standard_conforming_strings TO OFF;")
```

Comment:
Use single quotes for all strings (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).